src/Http/Http.Results/README.md

```diff
@@ -0,0 +1,13 @@
+# ASP.NET Core Http.Results
+
+Http.Results contains the in-framework implementations of the `IResult` interface returned from Minimal APIs route handler delegates, e.g. `OkHttpResult`, `NoContentHttpResult`, etc.
+
+## Development Setup
+
+The `Results<TResult1, TResult2, TResultN>` union types are generated. Modify and run the [ResultsOfTGenerator](tools/ResultsOfTGenerator/) tool to generate an updated `ResultsOfT.cs` class file.
+
+Run the following command in `src\Http\Http.Results\tools\ResultsOfTGenerator`:
+
+```
+dotnet run
+```
```

Comment:
Can we also add a test that fails if the generator and generated code get out of sync like [Kestrel's GeneratedCodeIsUpToDate test](https://github.com/dotnet/aspnetcore/blob/a450cb69b5e4549f5515cdb057a68771f56cefd7/src/Servers/Kestrel/Kestrel/test/GeneratedCodeTests.cs#L21-L77)?

Reply:
Done!